docs/techreview.ltx

```diff
@@ -0,0 +1,440 @@
+% assignment: http://eecs.oregonstate.edu/capstone/cs/capstone.cgi?hw=techreview
+
+\documentclass[10pt,draftclsnofoot,onecolumn,journal,compsoc]{IEEEtran}
+% for IEEEtran usage, see http://www.texdoc.net/texmf-dist/doc/latex/IEEEtran/IEEEtran_HOWTO.pdf
+
+\usepackage[margin=0.75in]{geometry}
+\usepackage{graphicx}
+\usepackage{caption}
+\usepackage{hyperref}
+\usepackage{enumerate}
+\usepackage{amssymb}
+
+\renewcommand{\linespread}{1.0}
+
+\title{COAL: Coal and Open-pit surface mining impacts on American Lands}
+\author{
+  \IEEEauthorblockN{Taylor Alexander Brown, Heidi Ann Clayton, and Xiaomei Wang \\ Group 18} \\
+  \IEEEauthorblockA{CS 461: Senior Capstone Fall 2016 \\ Oregon State University}
+  % TODO: group number
+}
+\date{}
+
+\IEEEtitleabstractindextext{
+  \begin{abstract}
+	Mining is known to cause environmental degradation, but software tools to identify mining impacts are lacking. Researchers studying this problem possess large imaging spectroscopy and environmental quality data sets as well as high-performance cloud-computing resources. This project provides a suite of algorithms using these data and resources to identify signatures of mining and correlate them with environmental impacts over time.
+  \end{abstract}
+}
+
+\begin{document}
+
+\maketitle
+\IEEEdisplaynontitleabstractindextext
+\IEEEpeerreviewmaketitle
+
+\newpage
+
+\tableofcontents
+
+\newpage
+
+% Introduction to entire tech review, including authorship of each section
+\section{Introduction}
+\label{sec:intro}
+This document is a review of technology and literature relevant to the COAL project. The technologies discussed in this paper correspond to tasks or subtasks identified in our requirements document \cite{requirements}: Develop software to process imagery, provide API documentation for developers, and provide a website that serves as the project homepage. At this point in the project our team is working with the client to become familiar with the data and the available literature, so some material is subject to change as our understanding improves.
+
+Sections \ref{subsec:featureextraction} and \ref{subsec:classification} were authored by Heidi who will take responsibility for the algorithms to identify and classify minerals. Section \ref{subsec:identifymining} was authored by Taylor on behalf of Xiaomei, who will take responsibility for the algorithms to identify regions of mining activity. Sections \ref{subsec:preprocessing}, \ref{subsec:correlate}, and \ref{subsec:changesovertime} were authored by Taylor who will take responsibility for the algorithms to correlate mining with environmental impact and to rank and document changes over time. Section \ref{subsec:apidoc} was written by Heidi who will take responsibility for the API documentation. Sections \ref{subsec:websitebackend} and \ref{subsec:websitefrontend} were written by Xiaomei who will take responsibility for implementing the backend and the frontend of the project homepage.
```

Comment:
in the current PDF of the document, some section references weren't generated correctly so they show up as question marks. rebuild may be necessary.